src/oracle/oracle.cairo

```diff
@@ -957,11 +957,20 @@ mod Oracle {
                 self, quote_data_type, aggregation_mode, sources
             );
 
-            let decimals = min(
+            let decimals = max(
```

Comment:
why did we switch to `max` ?

Reply:
it was just for testing purpose. The conclusion is that we don't even have to chose between a min and a max. Let's say we want to USD hop ETH/BTC, we will always have USD as quote, thus the decimals will be the USD decimals for both BTC/USD and ETH/USD (as we decided in the get_decimals definition). So the final version can just be 
```let decimals = IOracleABI::get_decimals(self, base_data_type)``. Do we agree on that ?

Reply:
makes sense